docs/content/grafana/README.md

```diff
@@ -1,16 +1,35 @@
 # Grafana
 
+<font size="4"> 
+
 [Grafana][1] is a multi-platform open source analytics and interactive visualization tool. It allows you to query, create dashboards, explore, and alert on metrics when connected to supported data sources.
 
-We use Grafana to visualize the metrics that we source from our [User Workload Monitoring][../uwm]. These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
+</font>
+
+<font size="3">
+
+We use Grafana to visualize the metrics that we source from our [User Workload Monitoring][10] (click to learn more). These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
 
 The dashboards can be altered, saved and reused as per your specific monitoring needs.
 
-Dashboards can be represented by the `GrafanaDashboard` custom resource or by their raw `JSON` files. You can find example monitoring dashboards defined in separate `JSON` files for each of the Operate First managed applications in our [apps repo][2]. For more information on how the Grafana dashboards are defined, please refer to [this documentation][3].
+Dashboards can be represented by the `GrafanaDashboard` custom resource or by their raw `JSON` files. You can find example monitoring dashboards defined in separate `JSON` files for each of the Operate First managed applications in our apps repo found [here][2]. For more information on how the Grafana dashboards are defined, please refer to [this documentation][3] (includes information regarding dashboard properties, creation, UIDs, error handling, plugins, dashboard discovery, datasource inputs, config map references, and dashboard folder support).
```

Comment:
> (includes information regarding dashboard properties, creation, UIDs, error handling, plugins, dashboard discovery, datasource inputs, config map references, and dashboard folder support) 

I don't think we should list individually the content addressed by grafana documentation. The average user will not really care about UIDs, error handling, etc. We just want to let them know about the existence of this document, and for further reading/information they can go there should they wish. We can add more clarification that its more than just for "defining" dashboards e.g: 

> For more information on how the Grafana dashboards are defined, please refer to [this documentation](https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md) ... 

to: 

> For any information on how to create, structure, and manage Grafana Dashboards see the official Grafana documentation [here](https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md)